### PR TITLE
Fix issues with a fresh installation not working on Ubuntu 22.04

### PR DESCRIPTION
**Pull request by raits**
Packages are available in the snapshot repository for testing. 

Resolves XRDSD-334.

### CODE REVIEW

File: packages/xtee-misp2-base/debian/changelog
Comment (petkivim) [inline packages/xtee-misp2-base/debian/changelog:4]: It would be good to add metadata about the maintainer and release date/time like the other entries (except the previous three), for example:

> -- NIIS [info@niis.org](mailto:info@niis.org) Wed, 06 Apr 2022 15:38:05 +0300

File: packages/xtee-misp2-orbeon/debian/changelog
Comment (petkivim) [inline packages/xtee-misp2-orbeon/debian/changelog:4]: It would be good to add metadata about the maintainer and release date/time like the other entries (except the previous three), for example:

> -- NIIS [info@niis.org](mailto:info@niis.org) Wed, 06 Apr 2022 15:38:05 +0300

File: packages/xtee-misp2-postgresql/debian/changelog
Comment (petkivim) [inline packages/xtee-misp2-postgresql/debian/changelog:4]: It would be good to add metadata about the maintainer and release date/time like the other entries (except the previous three), for example:

> -- NIIS [info@niis.org](mailto:info@niis.org) Wed, 06 Apr 2022 15:38:05 +0300

File: packages/xtee-misp2-postgresql/install-misp2-postgresql.sh
Comment (petkivim) [inline packages/xtee-misp2-postgresql/install-misp2-postgresql.sh:297]: I would change this a little bit to make it gender neutral:

> If user does not exist in the database then create them by running:

It's called "singular they":

https://en.wikipedia.org/wiki/Singular_they

Comment (petkivim) [reply packages/xtee-misp2-postgresql/install-misp2-postgresql.sh:297]: In future releases, we should review the whole documentation for similar cases.

**Review by petkivim — APPROVED** [review]

File: packages/xtee-misp2-application/debian/changelog
Comment (raits) [reply packages/xtee-misp2-application/debian/changelog:5]: Added, also updated the older entries since they were causing warnings.

File: packages/xtee-misp2-base/debian/changelog
Comment (raits) [reply packages/xtee-misp2-base/debian/changelog:5]: Added, also updated the older entries since they were causing warnings.

File: packages/xtee-misp2-orbeon/debian/changelog
Comment (raits) [reply packages/xtee-misp2-orbeon/debian/changelog:5]: Added, also updated the older entries since they were causing warnings.

File: packages/xtee-misp2-postgresql/debian/changelog
Comment (raits) [reply packages/xtee-misp2-postgresql/debian/changelog:5]: Added, also updated the older entries since they were causing warnings.

File: packages/xtee-misp2-postgresql/install-misp2-postgresql.sh
Comment (raits) [reply packages/xtee-misp2-postgresql/install-misp2-postgresql.sh]: Updated.

**Review by petkivim — APPROVED** [review]

**Pull request merged**
